src/components/WalletSuite/ConnectedWallet/Details/AdminLinks.tsx

```diff
@@ -25,7 +25,7 @@ export default function AdminLinks(props: WalletState) {
   const isLoading =
     isApLoading || isApFetching || isReviewLoading || isReviewFetching;
 
-  if (isLoading || !isApMember || !isReviewMember) return null;
+  if (isLoading || (!isApMember && !isReviewMember)) return null;
```

Comment:
should show even single
<img width="375" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/df68d8b2-0d4f-4abe-b078-7fa792921a59">